### PR TITLE
align text encoding with gui_element (DEV-1501)

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

File: rosetta.xml [hunk 1]
```diff
@@ -266,7 +266,9 @@
             <text permissions="prop-default" encoding="xml">
                 Tablet K.3375 in the British Museum is tablet 11 of the Epic of Gilgameš. It contains the very famous
                 story of the deluge. The tablet was found at Koyunjik in the Library of Ashurbanipal and it dates to
-                the 7<sup>th</sup> century BCE.
```
Comment (jnussbaum) [inline rosetta.xml:269]: I don't think that this autocorrection is good. Does your IDE let you revert this?

File: rosetta.xml [hunk 2]
```diff
@@ -524,9 +526,14 @@
         </text-prop>
         <text-prop name=":hasTranscription">
             <text permissions="prop-default" encoding="xml">
-                <sup>d</sup> GIŠ-gim₂-maš a-na ša₂-⸢šu⸣-ma MU-ra a-na <sup>m</sup> UD-ZI ru-u₂-qi₂
+                <sup>d</sup>
```
Comment (jnussbaum) [inline rosetta.xml:529]: Again, I would revert this

File: rosetta.xml [hunk 3]
```diff
@@ -752,112 +785,197 @@
             <text permissions="prop-default" encoding="utf8">Liberté</text>
         </text-prop>
         <text-prop name=":hasTranscription">
-            <text permissions="prop-default" encoding="utf8">
+            <text permissions="prop-default" encoding="xml">
                 Sur mes cahiers d’écolier
+                <br />
```
Comment (jnussbaum) [inline rosetta.xml:790]: I would put the `<br />` on the old line in order to economize space, but it's a question of style

File: rosetta.xml [hunk 4]
```diff
@@ -986,39 +1104,70 @@
             <text permissions="prop-default" encoding="utf8">Стрекоза и Муравей</text>
         </text-prop>
         <text-prop name=":hasTranscription">
-            <text permissions="prop-default" encoding="utf8">
+            <text permissions="prop-default" encoding="xml">
                 Стрекоза и Муравей
-
+                <br />
```
Comment (jnussbaum) [inline rosetta.xml:1109]: same here - feel free to put the `<br />` on the old line, or leave them as they are

File: rosetta.xml [hunk 5]
```diff
@@ -1697,18 +1859,30 @@
             <text permissions="prop-restricted" encoding="utf8">The text above has no "permissions" attribute, so only the System Administrator can see it!</text>
             <text permissions="prop-restricted" encoding="utf8">Demonstration of multiple cardinalities: show how 0-n cardinalities work</text>
             <text permissions="prop-restricted" encoding="xml">
-                Even <strong>XML-enriched text</strong> works in a "gui_element": "SimpleText", but it is better to use
-                "Richtext".<br />
-                This is <sup>supralinear text</sup> and has a <a class="salsah-link" href="IRI:obj_0:IRI">link</a> to
+                Even
```
Comment (jnussbaum) [inline rosetta.xml:1862]: Here again, I would revert it

**Comment by jnussbaum** [comment]
I think you forgot to change two occurrences: 
 - (new) line 743 should be utf8
 - (new) line 2124 should be xml

The rest of my comments is of cosmetic nature.

**Comment by BalduinLandolt** [comment]
@jnussbaum no, you're right. I was about to correct them, I just hit "re-request" too early. Sorry about that. Now they are fixed

**Review by jnussbaum — APPROVED** [review]

**Pull request merged**